.github/workflows/build.yaml

```diff
@@ -46,14 +46,18 @@ jobs:
         with:
           apt-get: git python3-jinja2 python3-yaml python3-deepdiff python3-git python3-github python3-requests xmldiff
       # TODO: Use action's checkout along with --local and --repository options of ctf
+      # See: https://github.community/t/how-to-commit-to-two-branches-from-an-action/17713/4
       #- name: Checkout
-      #  uses: actions/checkout@v2
+      #  uses: actions/checkout@v1
       - name: Checkout (CTF)
         uses: actions/checkout@v2
```

Comment:
The comment above talks about actions/checkout@v1, why v2 is here?

Reply:
The checkout@v1 is the only way to get multiple branches at the same time (which will allow us to switch to action-only workflow for content filter).